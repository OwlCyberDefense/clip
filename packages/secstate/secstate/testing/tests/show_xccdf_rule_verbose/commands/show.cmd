/usr/sbin/secstate show PassComp PassComp-R-2-2 -v
