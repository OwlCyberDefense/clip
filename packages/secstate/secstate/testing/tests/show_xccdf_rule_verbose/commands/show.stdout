PassComp-R-2-2:
	Title:  'Min. Length'
	Description:  Password has a minimum length.
	Selected:  True
	Member of PassComp-G-2-3
	Referenced Definitions:
		oval:com.tresys.oval.rhel:def:1001
